arches_component_lab/src/arches_component_lab/widgets/utils.ts

```diff
@@ -67,3 +67,64 @@ export function convertISO8601DatetimeFormatToPrimevueDatetimeFormat(
 
     return { dateFormat: processedFormat, shouldShowTime: containsTime };
 }
+
+export function getUpdatedComponentPath(
+    deprecatedComponentPath: string,
+): string {
+    const deprecatedComponentToUpdatedComponentPathMap: Record<string, string> =
+        {
```

Comment:
missing the following widget references
```suggestion
            "views/components/widgets/switch":
                "arches_component_lab/widgets/SwitchWidget/SwitchWidget",
            "views/components/widgets/iiif":
                "arches_component_lab/widgets/IIIFWidget/IIIFWidget",
```